File: weekThreeDistributedApplications/day5HTMLandSQL/Day4_OOP_HTML5/ObjectOrientedProgramming/Pizza.cs
using System;

namespace ObjectOrientedProgramming
{
    class Pizza : Food
    {
        

        //Methods
        //Constructor special method with same name as the class no return type 
        // next to its name
        private string crust = "";
        public Pizza():base()
        {
            
        }
        public Pizza(string name, double price, double cost,string crust):base(name,price,cost)
        {
           this.crust = crust;
        }

       
        

    }


}

File: weekFourProjects/Pizza/pizza.js
class Pizza {
    constructor(base, topping, size, price){
        this.base = base
        this.topping = topping
        this.size = size
        this.price = price
    }
    getBase(){
        return this.base
    }
}

myPizza = new Pizza("Thin", "Ham and Pineapple", 16, 12.99)
m = myPizza.getBase()
console.log(m)